### PR TITLE
Add login and character map move functionality

### PR DESCRIPTION
**Pull request by dracc**
This became quite a big change.

### Changes proposed by this PR:
- Add login form connected to game account credentials
- Switch `Account` tab into `Map Switch` when user is logged in
- Add functionality to allow map switches for offline characters

### Known problems
- ~~JSON data in `chardb->characters->entitydata` lacks whitespace after a map move. Shouldn't matter, but I'm not sure how picky the cerealizer in SEGS is.~~

### CODE REVIEW

**Comment by dracc** [comment]
I've now confirmed that the client and/or server does not care about the lack of whitespace in the binary blob named `entitydata`.

But maybe, just maybe, we'd want to reset coordinates on city move, as moving currently often results in a need to use `/stuck`.

File: scripts.js [hunk 1]
```diff
@@ -83,42 +83,200 @@ goSlogs = function(){
...
+                  window.location.reload();
+              }
+          });
+    return false;
+}
+
+signOut = function(){
+    var signingout = "p";
+    fetch("/logout.php",
+          {method: 'POST',
```
Comment (dracc) [inline scripts.js]: This should be GET.

File: scripts.js [hunk 2]
```diff
@@ -83,42 +83,200 @@ goSlogs = function(){
...
+          }).then(function(data){
+              console.log(data);
+              resultbox.innerHTML=data.retmsg;});
 }
 
 accsUpdate = function(){
     var ua = document.getElementById("numaccs");
     var cc = document.getElementById("numchars");
-    fetch(location.protocol + "//" + window.location.hostname + "/acc_count.php", {method: 'GET'
+    fetch("/acc_count.php", {method: 'GET'
```
Comment (dracc) [inline scripts.js]: Please linebreak after filename to stay consistent with the other fetches.

**Comment by dracc** [comment]
Todays changes:
- Favicon added
- character position is reset to [0,0,0] on city switch
- confirmation text on successful switch
Screenshot

**Comment by mobbyg** [comment]
Looks good. I can’t test now as I’m at work. But I’ll push this and we can address anything that crops up in a future PR.

**Pull request merged**